go.mod

```diff
@@ -39,6 +39,7 @@ require (
 	github.com/zclconf/go-cty v1.10.0
 	go.opentelemetry.io/otel v1.14.0
 	go.opentelemetry.io/otel/trace v1.14.0
+	golang.org/x/mod v0.12.0
```

Comment:
If we don't need to bump to v0.12.0, can you just keep v0.9.0? But if needed then can you do the vendoring in a dedicated commit?

Reply:
Yea I can throw this back down to `v0.9.0`. I figured it was likely at `v0.9.0` because it was an implicit dependency, but it's likely better to just do any version upgrades as a different PR anyway.